src/components/terms-and-conditions.tsx

```diff
@@ -0,0 +1,151 @@
+                            <li>
+                              for any loss that may result from advice or
+                              instructions concerning repairs,
+                            </li>
+                            <li>
+                              for the loss of items handed over for repair,
+                            </li>
+                            <li>for indirect or consequential loss,</li>
+                            <li>
+                              for any other kind of loss resulting from work
+                              performed at Repair Lab.
+                            </li>
+                          </ol>
+                        </li>
+                        <li>
+                          A voluntary donation is always appreciated, to help
+                          fund future Repair Labs.
+                        </li>
+                        <li>
+                          Any use of new materials such as leads, plugs, fuses,
+                          etc. will be paid for separately.
+                        </li>
+                        <li>
+                          Visitors offering broken items for repair do so at
+                          their own risk.
+                        </li>
+                        <li>
+                          Volunteer Mending Mentors offer no guarantee for the
+                          repairs carried out with their help and are not liable
+                          if objects that are repaired at Repair Lab turn out
+                          not to work properly at home.
+                        </li>
+                        <li>
+                          Repair Volunteers are entitled to refuse to repair
+                          certain objects.
+                        </li>
+                        <li>
+                          Repair Volunteers are not obliged to reassemble
+                          disassembled appliances that cannot be repaired.
+                        </li>
+                        <li>
+                          Participants at Repair Lab are solely responsible for
+                          the tidy removal of broken objects that could not be
+                          repaired.
+                        </li>
+                        <li>
+                          Photographs or videos are taken during repair sessions
+                          and may be used for online or in-print news and
+                          promotional materials. If you do not wish to have your
+                          photograph used, please advise the volunteer taking
+                          the photos or the organizers.
+                        </li>
+                      </ol>
+                    </p>
+                  </div>
+
+                  <div className="mt-8 flex justify-center">
+                    <Button onClick={handleshowPopupChange}>Close</Button>
+                  </div>
+                </Dialog.Panel>
```

Comment:
Maybe add an `Accept & Close` button that would close the modal and tick the checkbox.
Move the `Close` button to the top right and replace it with an `X`.
@BoboJeager @dct0 What do you think?